portfolio/src/main/webapp/js/script.js

```diff
@@ -95,4 +121,132 @@ function deleteComment(message) {
   const params = new URLSearchParams();
   params.append('id', message.id);
   fetch('/delete-comment', {method: 'POST', body: params});
+  getText();
+}
+
+
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const nightmode = [
+      {elementType: 'geometry', stylers: [{color: '#242f3e'}]},
```

Comment:
2-space indent instead of 4

https://google.github.io/styleguide/jsguide.html#formatting-block-indentation